.github/workflows/stale-backtrace-issues.yml

```
@@ -0,0 +1,17 @@
+name: 'Manage stale PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v7
+        with:
+          days-before-issue-stale: 60
+          days-before-issue-close: 7
+          only-issue-labels: 'backtrace.io'
+          stale-issue-label: 'stale-backtrace'
+          stale-pr-message: 'This issue is stale because it has been open for 60 days with no activity. Remove stale label or comment or this will be closed in 1 day.'
```

"1 day" doesn't match the `days-before-issue-close` value of 7 that's set earlier.

---

This matches the other requested change. "in 1 day" can be changed to "tomorrow" if needed.
```suggestion
          stale-pr-message: 'This issue is stale because it has been open for 60 days with no activity and will be closed in 1 day. To keep it open, remove the "stale" label or leave a comment.'
```

---

It's meant to be 1 day, 61 days is more than enough time for those.